go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.5.2
 	github.com/hashicorp/go-slug v0.4.1
 	github.com/hashicorp/go-uuid v1.0.1
-	github.com/hashicorp/jsonapi v0.0.0-20210326175023-166ae20eb0d1
+	github.com/hashicorp/jsonapi v0.0.0-20210401204629-95d572587ba2
```

Comment:
Did we want to bump this to latest here, or do it elsewhere?

Reply:
This was for the links work, but i'll update it to include the recent timestamps addition